geode-core/src/main/java/org/apache/geode/cache/query/security/RegExMethodAuthorizer.java

```diff
@@ -85,7 +85,8 @@
    *
    * @return an unmodifiable view of the regular expressions used to configure this authorizer.
    */
-  public Set<String> getAllowedPatterns() {
+  @Override
+  public Set<String> getParameters() {
```

Comment:
Not quite related to this diff... but the javadoc mentions that the returned view/set is unmodifiable.  Should this be wrapped into an actual unmodifiable collection instead of the real set?  I guess the same question would be for all of the getParameter methods.  Or we modify the javaDoc.

Reply:
When the `allowedPatterns` set (and the `allowedPackages` set in the JavaBeanAccessorMethodAuthorizer) are assigned, it's as a `Collections.UnmodifiableSet` object, and there are unit tests to confirm that the output of these methods are unmodifiable.